content/500-support/100-help-articles/200-working-with-many-to-many-relations.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: 'Working with many-to-many relations with Prisma'
+metaTitle: 'Working with many-to-many relations with Prisma'
+metaDescription: 'Learn how you can model and query implicit and explicit many-to-many relations with Prisma'
+---
+
+## Problem
+
+Lots of users have queries about can one model many-to-many relations and query for the data using Prisma. Here are two examples one implicit and one explicit to show how you can handle these with Prisma.
+
+## Solution
+
+### Implicit relations
+
+This is a type of many-to-many relation where Prisma handles the join table internally. A basic example for an implicity many-to-many relation would look like this:
```

Comment:
Our official term for join table is [relation table](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#relation-tables):

```suggestion
This is a type of many-to-many relation where Prisma handles the [relation table](../../concepts/components/prisma-schema/relations#relation-tables) internally. A basic example for an implicit many-to-many relation would look like this:
```